beacon-chain/sync/data_columns_sampling.go

```diff
@@ -522,7 +522,7 @@ func verifyColumn(
 	}
 
 	// Filter out columns which did not pass the KZG inclusion proof verification.
-	if err := blocks.VerifyKZGInclusionProofColumn(roDataColumn.DataColumnSidecar); err != nil {
+	if err := blocks.VerifyKZGInclusionProofColumn(roDataColumn); err != nil {
```

Comment:
Isn't the whole idea to use the new verification pipeline everywhere? (Included here, and in initial sync?)

Reply:
Yes that is the ultimate idea, but I didn't want to replace everything in this PR itself. The focus of this PR is simply for gossip validation rules. In a follow up PR i will add for pending queues + initial sync